ecosystem/typescript/sdk/examples/typescript/gas_fee_payer.ts

```diff
@@ -16,6 +16,7 @@ import {
 } from "aptos";
 import { NODE_URL, FAUCET_URL } from "./common";
 import { assert } from "console";
+import * as Gen from "../../src/generated";
```

Comment:
@0xmaayan , is there a better way to import this than using a relative path like this?

Reply:
You can import 'Type' and then use it 'Type.<thetypeyouneed>'

Reply:
Applied. Thank you!